sdk/src/types/block/output/nft.rs

```diff
@@ -257,20 +256,9 @@ impl NftOutputBuilder {
         Ok(output)
     }
 
-    ///
-    pub fn finish_with_params<'a>(self, params: impl Into<ValidationParams<'a>> + Send) -> Result<NftOutput, Error> {
-        let output = self.finish()?;
-
-        if let Some(token_supply) = params.into().token_supply() {
-            verify_output_amount_supply(output.amount, token_supply)?;
-        }
-
-        Ok(output)
-    }
-
     /// Finishes the [`NftOutputBuilder`] into an [`Output`].
-    pub fn finish_output<'a>(self, params: impl Into<ValidationParams<'a>> + Send) -> Result<Output, Error> {
-        Ok(Output::Nft(self.finish_with_params(params)?))
+    pub fn finish_output<'a>(self) -> Result<Output, Error> {
```

Comment:
no need for the lifetime param any longer

Reply:
Removed